README.md

```diff
@@ -353,6 +353,41 @@ Releasing
 
 [How to Release Apache Fineract](https://cwiki.apache.org/confluence/x/DRwIB) documents the process how we make the source code that is available here in this Git repository into a binary release ZIP available on http://fineract.apache.org.
 
+Before you use Gradle to create a release you need to make sure that your GPG is properly setup and that you have set the following properties in your ~/gradle/gradle.properties:
+```
+signing.gnupg.keyName=7890ABCD
+signing.gnupg.passphrase=secret
+```
+
+IMPORTANT: Do not set your GPG secrets in one of the project gradle.properties and double check that you are not accidentally committing them to Git.
+
+NOTE: Let's assume your GPG key ID would be "ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890ABCD" then you have to use the last 8 characters (i. e. "7890ABCD") for the signing plugin property "signing.gnupg.keyName". 
+
+Execute the following task to create a distribution with an ASCII armored signature (.asc) and a SHA512 checksum file (.sha512):
+```
+./gradlew -Pfineract.release clean build 
+```
```

Comment:
Would it be cleaner to have a specific distribution task that does all the distribution activities (package and signing etc), rather than relying on the build task and a new profile?

Reply:
I had that conversation with  @edcable and it was preferred to include this "naturally" (i. e. without a specific task) in the build process. Note: the release artifacts are only created when "-Pfineract.release" is provided on the command line which is pretty explicit. For the next release I intend to add even more features here (publishing client jar on Maven Central, uploading distribution to Apache, maybe publishing release Confluence page).

Reply:
OK - if it's been decided, then that's fine. Personally I would have preferred that all different activities are triggered as tasks: it's a bit confusing now that all other activities are triggered as tasks and only these with a profile. 

Also as far as I know, you can only have one profile active - correct? So if in the future we want to have different types of builds activated as profiles, you can't create distributions of them.

Reply:
My first "reflex" was also to add an explicit task, but have to say that I'm not necessarily opposed to the way it's now (aka as a side effect of the distribution task). Having said that: given the short time frame I made a couple of lone decisions so I think it makes a lot of sense to present this to a broader audience (mailing list) and let them have a say how this should look like for the next release.

For now I just wanted to have something that helps me during the release.

I'm not sure how this (creating the release via a command line parameter) would impede different profiles? Let's discuss this on the mailing list or maybe in a separate Jira ticket?